apps/worker/src/types/env.ts

```
@@ -6,7 +6,7 @@ export interface Env {
   R2: R2Bucket;
 
   // Queue
-  NOTE_QUEUE: Queue;
+  NOTE_QUEUE: Queue<any>;
```

ESLint

[error] 'Queue' is not defined.

(no-undef)